digitalforms-api/src/main/resources/application.properties

```diff
@@ -20,3 +20,8 @@ management.endpoint.beans.cache.time-to-live=10s
 vips.client.basePath=${DIGITALFORMS_VIPSORDS_BASEPATH}
 vips.client.username=${DIGITALFORMS_VIPSORDS_USER} 
 vips.client.password=${DIGITALFORMS_VIPSORDS_PASSWORD}
+
+# Digital Forms ORDS client properties
```

Comment:
Good. Does Ebenezer need to know about the new variables?

Reply:
I have given him the details needed to update his config files, thank you :)

Reply:
Perfect, You're ahead of me :-)